www/templates/diary/infinite_scroll_list.html

```diff
@@ -41,6 +41,7 @@ <h2 class="hr-lines" translate> {{trip.display_date}}</h2>
                 this and implement some sort of horizontal line between trips-day-changing later on-->
 
                 <infinite-scroll-trip-item trip="trip" map-limiter="mapLimiter"></infinite-scroll-trip-item>
+                <infinite-scroll-place-item ng-if="showPlaces" trip="trip"></infinite-scroll-trip-item>
```

Comment:
pro/con for having the `ng-if` here versus in the `place_list_item.html`. I have a mild preference to keep the logic in the directive instead of the list. In either case, I would like to document the rationale behind the decision for the future.

Reply:
Up to you. I prefer to hide/show elements at their root because I don't want empty containers in the DOM

I can move the `ng-if` to `place_list_item.html`, but when the config is set `false` I believe we will have an empty `<infinite-scroll-place-item>`.

Reply:
I'm fine with leaving it here, but can you please add a comment with this rationale, so we don't have somebody looking at this 5 years from now saying (why? why?)